module_1/src/MovieListPage/MovieListPage.js

```diff
@@ -0,0 +1,146 @@
+import React, {useState, useEffect} from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
+import SearchForm from '../SearchForm/SearchForm';
+import MovieTile from '../MovieTile/MovieTile';
+import SortControl from '../SortControl/SortControl';
+import MovieDetails from '../MovieDetails/MovieDetails';
+import './MovieListPage.css';
+
+function MovieListPage (props) {
+    const [movies, setMovies] = useState(null);
+    const [totalMovieAmount, setTotalMovieAmount] = useState(null);
+    const [searchQuery, setSearchQuery] = useState('');
+    const [isMovieSelected, setIsMovieSelected] = useState(false);
+    const [movieDetails, setMovieDetails]= useState({});
+    const options = ['RELEASE DATE', 'TITLE'];
+    const [sortCriterion, setSortCriterion] = useState('release_date');
+    const mainGenres = ['ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME'];
+    const [isGenreActive, setIsGenreActive] = useState('ALL');
+    const navigate  = useNavigate();
+    const location = useLocation();
+
+    useEffect(()=> {
+        fetch(`http://localhost:4000/movies?sortBy=${sortCriterion}&sortOrder=asc`)
+        .then((response) => response.json())
+        .then ((json)=> {
+            setMovies(json.data);
+            setTotalMovieAmount(json.data.length)
+        });
+    }, [sortCriterion]);
+
+    useEffect(()=> {
+        if (location.pathname === '/' ) {
+            setIsMovieSelected(false);
```

Comment:
Looks like there is no button on UI to switch back to search component
Guess it should be there
![image](https://github.com/VolhaChubakova/React/assets/30400314/23ab7ec9-49d9-4fd6-b5a8-256841fb932b)